### ISSUE TITLE
Create multistage stratified sample from `apipop`

### ISSUE THREAD

**Issue by smishr** [xKDR/Survey.jl#273]
Multistage stratified surveys have stratas and clustering.

I suggest drawing a sample from apipop with strata and clustering and export it in `assets/`. Use that data for testing the various algorithms in Survey.jl

**Comment by ayushpatnaikgit** [comment]
@britgyu wants to do this. @smishr are you confident this is a “good first issue”?

**Comment by smishr** [comment]
> @britgyu wants to do this. @smishr are you confident this is a “good first issue”? 

yes. they should give link to the code they used for drawing the multi stage sample, so we can double check and tweak if needed

**Comment by ayushpatnaikgit** [comment]
I remember there being a page where the author using dplyr to create sample datasets from apipop. Do you remember?

**Comment by smishr** [comment]
yes i have that code somewhere in my testing R file

**Comment by ayushpatnaikgit** [comment]
Can you dig out the link? I remember it was using `srvyr`

**Comment by smishr** [comment]
Click on drop down arrow where it says "Show R Code to make data" on this [blog post](https://www.practicalsignificance.com/posts/understanding-the-survey-packages-recursive-algorithm/)
